events_protocol/core/logging/__init__.py

```diff
@@ -51,19 +51,27 @@ def __init__(self, klass=None):
     def log(self, level, msg, *args, **kwargs):
         if self.isEnabledFor(level):
             event_context = EventContextHolder.get()
-            _msg = dict(
-                severity=logging.getLevelName(level),
-                logmessage=msg,
+            event_payload = dict(
                 EventID=event_context.id,
                 FlowID=event_context.flow_id,
                 UserId=event_context.user_id,
                 UserType=event_context.user_type,
                 Operation="{}:v{}".format(event_context.event_name, event_context.event_version),
                 logger=self.klass,
                 LoggerName=self.logger.name,
-                logdate=dt.utcnow().isoformat(),
                 ApplicationVersion=self.version,
             )
+            event_payload = dict(filter(lambda item: item[1] is not None, event_payload.items()))
+            event_payload = dict(
+                filter(lambda item: item[1] != "None:vNone", event_payload.items())
+            )
```

Comment:
Creio que é melhor juntar os dois filtros em um

Reply:
Pra falar a verdade, to me perguntando aqui se vale a pena ter esses filtros